ui.apps/src/main/content/jcr_root/apps/core/fd/components/aemform/v1/aemform/aemform.html

```diff
@@ -42,10 +42,17 @@
 <sly data-sly-use.clientLib="${'/libs/granite/sightly/templates/clientlib.html'}"/>
 <sly data-sly-test.useIframe="${form.useIframe != 'false'}">
     <sly data-sly-test.formLocaleString="${form.locale || 'en'}"/>
-    <iframe class="cmp-aemform__iframecontent"
-            src="${request.contextPath}${resource.path}.iframe.${formLocaleString}.html?dataRef=${request.requestParameterMap['dataRef'] != null ? request.requestParameterMap['dataRef'][0].toString : '' @ context='attribute'}${!wcmmode.edit ? '&wcmmode={0}':'' @ format=[wcmmode.toString]}"
-            stye="width:100%;"
-            data-form-page-path="${form.formEditPagePath}"></iframe>
+    <sly data-sly-set.dataRef="${request.requestParameterMap['dataRef'] != 'unsafe' ? request.requestParameterMap['dataRef'][0].toString : ''}"/>
```

Comment:
unsafe is a context, but you are checking for unsafe in the value of dataRef which is wrong

Reply:
but Rishi by setting context as unsafe it will disable the XSS protection isn't it? or shall I proceed with @context='unsafe' ?